source/index.md

```diff
@@ -469,134 +469,134 @@ Every cleric can choose prayers from the basic prayers and from the prayer list
 # Core mechanics
 
 ## Game Structure
-A game of **Dead Weight** runs a bit differently then most tabletop RPGs, it follows a strong structure, with two main states.
+A game of **Dead Weight** runs a bit differently than most tabletop RPGs. It follows a strong structure, with two main states.
 
 - [Expedition](#expedition-phase)
 - [Interlude](#interlude)
 
-The game usually starts on the **Expedition**, with the adventurers raiding a dungeon or some other terrible place in the **Blacksalt Isle**.
+The game usually starts in the **Expedition** phase, with the adventurers raiding a dungeon or some other terrible place in the **Blacksalt Isle**.
 
 ## Dice Notation
-Like many tabletop RPGs, **Dead Weight** uses dice notation to specify when and how you should roll your dice. Most of dice roll are common six-sided dice, unless specified otherwise.
+Like many tabletop RPGs, **Dead Weight** uses dice notation to specify when and how you should roll your dice. Most die rolls are common six-sided dice, unless specified otherwise.
 
-When you see **d4**, **d6**, **d8**, **d10**, **d12** and **d20** this is referring to a die of that size, d4 is a four-sized dice and d20 is a twenty-sided dice. Let's review all styles of dice notations we will use through this book:
+When you see **d4**, **d6**, **d8**, **d10**, **d12** and **d20**, this is referring to a die of that size; d4 is a four-sided die and d20 is a twenty-sided die. Let's review all styles of dice notations we will use in this book:
 
 ### Adding dice (2d6) {.unlisted}
-And when you see **3d6** means to roll **three six-sided dice** and add the result.
+When you see **3d6**, that means roll **three six-sided dice** and add the result.
 
 ### Usage dice (Ud8) {.unlisted}
-Usage dice is a weird but very useful concept. Usage dice are used for tracking how many consumables a character has, like **Trowing Daggers(Ud8)** has a usage dice of a d8. Every time the consumable is used the usage dice should be rolled, if the result is a 1 or 2, that dice is downgraded to the smallest die in the following chain:
+Usage dice track consumable use. For instance, **Throwing Daggers (Ud8)** have a d8 usage die. Every time the consumable is used, the usage dice are rolled: if the result is a 1 or 2, that die is downgraded to the (next?) smallest value in the following chain:
```

Comment:
Wasn't sure if the chain should exist for a reason if it's not going to the next smallest, but always the smallest. I think next was intended here, will add if you give the go-ahead.

Reply:
yes, next was intended!
```suggestion
Usage dice track consumable use. For instance, **Throwing Daggers (Ud8)** have a d8 usage die. Every time the consumable is used, the usage dice are rolled: if the result is a 1 or 2, that die is downgraded to the next smallest value in the following chain:
```